src/components/EditCoursePage/CollapsibleCourseRun.jsx

```diff
@@ -178,6 +196,13 @@ class CollapsibleCourseRun extends React.Component {
     if (status === PUBLISHED) {
       return 'Publish Run';
     }
+    if (status === REVIEWED) {
+      if (initialRuns && currentRuns) {
+        return isNonExemptChanged(initialRuns, currentRuns, key) ||
```

Comment:
Nit: Change `initialRuns` -> `initialValues.course_runs` and `currentRuns` -> `currentFormValues.course_runs`

It would at least let us lose those six lines above from deconstructing now that we already have to parse out currentFormValues and initialValues